automation/src/test/resources/data/parquet/generate_parquet_timestamp_list.py

```diff
@@ -0,0 +1,45 @@
+from pyspark.sql import SparkSession
+from pyspark.sql.functions import expr
+from pyspark.sql.types import IntegerType
+import shutil,os
```

Comment:
Style-nit: 

```suggestion
import shutil, os
```

I don't think the team has an existing convention for Python code style, but I generally run [`black`](https://github.com/psf/black) on all my Python code so that it is consistent and automatically formatted for me.

Reply:
formatted using `black`